nixos/modules/services/web-servers/pomerium.nix

```diff
@@ -0,0 +1,665 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  goDuration = types.mkOptionType {
+    name = "goDuration";
+    description = "Go duration (https://golang.org/pkg/time/#ParseDuration)";
+    check = x: types.str.check x && builtins.match "(-?[0-9]+(\.[0-9]+)?(ns|us|µs|ms|s|m|h))+" x != null;
+    inherit (types.str) merge;
+  };
+in
+{
+  options.services.pomerium = {
+    enable = mkEnableOption "the Pomerium authenticating reverse proxy";
+
+    bindLowPort = mkOption {
```

Comment:
Is there no way to set the ports in the configuration file? :open_mouth:

Reply:
There is, but I couldn't be bothered to parse it out of the Go-style bind address.

Reply:
Unfortunate. Maybe you can add a `port` like option which sets the ports, configured capabilities, optionally opens firewall, etc...

Reply:
I think I'll just remove this as an option and always assume the end user is going to bind to a low port, especially if we're going with the monolithic invisible settings approach.